apps/admin/app/api/partners/moxie/route.ts

```diff
@@ -1,42 +1,36 @@
+import { log } from '@charmverse/core/log';
 import { prisma } from '@charmverse/core/prisma-client';
 import { currentSeason, getLastWeek } from '@packages/scoutgame/dates';
 import { airstackRequest } from '@packages/scoutgame/moxie/airstackRequest';
-import { getMoxieFanToken } from '@packages/scoutgame/moxie/getMoxieFanToken';
 import { uniq } from 'lodash';
-
-import { respondWithTSV } from 'lib/nextjs/respondWithTSV';
+import { v4 } from 'uuid';
 
 export const dynamic = 'force-dynamic';
 
-type MoxieBonusRow = {
-  'Builder FID': number;
-  'Builder path': string;
-  'Builder event': string;
-  'Scout FID': number;
-  'Scout email': string;
-  'Scout path': string;
-};
-
 export async function GET() {
   const lastWeek = getLastWeek();
   const builders = await prisma.scout.findMany({
     where: {
-      builderStatus: 'approved'
+      builderStatus: 'approved',
+      farcasterId: {
+        not: null
+      },
+      hasMoxieProfile: true,
+      userWeeklyStats: {
+        some: {
+          week: lastWeek,
+          gemsCollected: {
```

Comment:
the info page said that the builder must have atleast 1 gems earned for the week